pkg/maps/metricsmap/metricsmap.go

```diff
@@ -188,6 +204,16 @@ func newMetricsMapCollector(logger *slog.Logger) prometheus.Collector {
 			"Total forwarded bytes, tagged by ingress/egress direction",
 			[]string{metrics.LabelDirection}, nil,
 		),
+		fragNeededCountDesc: prometheus.NewDesc(
+			prometheus.BuildFQName(metrics.Namespace, "", "frag_needed_count_total"),
+			"Total number of icmp fragmentation-needed or icmpv6 packet-too-big packets",
```

Comment:
Maybe name this `icmp_frag_needed_count` instead? The `count` and `total` suffixes are redundant. And the name suggests it indicates all packets that needed fragmentation, rather than number if ICMP messages.

Reply:
Makes sense, improved it a bit to make the purpose more clear.